### ISSUE TITLE
2021-12-4

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#675]
81How to unzip tar.gz file in ubuntu? tar -xf archive.tar.gz ; (2021-11-23)
82Transfer it to our own server; done; download cifar100; send it to server; unzip it; done; (2021-11-23)
83What is L2 loss? L1 Loss function stands for Least Absolute Deviations. Also known as LAD. L2 Loss function stands for Least Square Errors. Also known as LS. (2021-11-23)
84Add L1 loss to database; done; add L2 loss to database; done; (2021-11-23)
85python train.py test multi_resnet18_kd  --data-dir /home/gaoye/pytorch-be-your-own-teacher/cifar-100-python --resume /home/gaoye/pytorch-be-your-own-teacher/checkpoint
86python train.py train multi_resnet18_kd --data-dir /home/gaoye/pytorch-be-your-own-teacher/cifar-100-python ; batch_size 128 is too large, try 64; 64 is too large, try 32; 
87python train.py train multi_resnet18_kd --data-dir /root/pytorch-be-your-own-teacher 
88Be Your Own Teacher: Improve the Performance of Convolutional Neural Networks via Self Distillation
89Linfeng Zhang, Jiebo Song, Anni Gao, Jingwei Chen, Chenglong Bao, Kaisheng Ma
